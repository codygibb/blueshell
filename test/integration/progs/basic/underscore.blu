func hello() {
  print "in hello"
  return "ignored"
}

err := 1337
_, err = try $> ls
print err
_, diff_err := try $> ls
_, diff_err = try $> ls
_ = hello()
_ := hello()

print diff_err
print _
