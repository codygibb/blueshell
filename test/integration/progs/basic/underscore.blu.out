0
in hello
in hello
0
error@15: Var_not_found _
